nengo_bones/templates/.travis.yml.template

```diff
@@ -0,0 +1,131 @@
+# Automatically generated by nengo-bones, do not edit this file directly
+# Version: {{ version }}
+
+language: python
+python:
+  - "3.6"
+notifications:
+  email:
+    on_success: change
+    on_failure: change
```

Comment:
What are the implications of changing all of these notification emails?

Reply:
This will send an email to the author of the commit, if that commit causes the build to start failing or start passing.  So e.g. if you make a PR and everything continues to build successfully as you'd expect, no email.  Or if you're working on a WIP branch and the build is failing all the time, you'll just get one email letting you know the build is broken, rather than a new email for each broken commit.

Before in nengo core and nengo loihi it was set up so that emails always went to Trevor, but it seems more useful to have them go to the author of the responsible commit.

Reply:
Most important change in this repo to me :heart: